compiler/src/iree/compiler/Codegen/LLVMCPU/LLVMCPULowerExecutableTarget.cpp

```diff
@@ -128,13 +128,7 @@ void LLVMCPULowerExecutableTargetPass::runOnOperation() {
   }
   case IREE::Codegen::DispatchLoweringPassPipeline::CPUDoubleTilingExpert: {
     TilingConfig tilingConfig = getTilingConfigForPipeline(funcOp);
-    addMultiTilingExpertPassPipeline(pipeline, tilingConfig, pipelineOpts);
-    break;
-  }
-  case IREE::Codegen::DispatchLoweringPassPipeline::
-      CPUDoubleTilingPeelingExpert: {
-    TilingConfig tilingConfig = getTilingConfigForPipeline(funcOp);
-    pipelineOpts.enablePeeling = true;
+    pipelineOpts.enablePeeling = isLoopPeelingEnabled(funcOp);
```

Comment:
Can we move it to where we are updating `pipelineOpts` (i.e., line 100)? Then we don't need it the same code in adding `CPUConvTileAndDecomposeExpert` passes pipeline.

Reply:
Yup, good suggestion, thanks!